changelogs/unreleased/gh-8861-vshard-options.md

```diff
@@ -1,3 +1,4 @@
 ## feature/config
 
 * Most of vshard options are now added in the config (gh-8861).
+* Added initial support of vshard (gh-8861).
```

Comment:
> 4) The router can automatically find all masters, but once all masters
> are found, any changes to the masters will be ignored until
> vshard.router.cfg() is called manually.

TBH, I don't get why it is so (maybe we talked around, but I forgot). Is it some vshard problem? Is it tracked in an issue?

Reply:
I am not sure if this actually should work. And isn't this part of one of master auto-discovering issues?

Reply:
In fact, I just want to understand what is going on. My thought was that everything should be fine with master discovery on routers. Now I'm not sure and I afraid that we can find more problems if we look inside.

Reply:
Let's elaborate it later.